library/errata_tool_product_version.py

```diff
@@ -112,6 +112,16 @@
        - What are the consequences of a completely empty brew_tag list? This
          might be answered in CLOUDWF-2.
      required: true
+   suppress_push_request_jira:
+     description:
+       - Set to true to suppress creating push request jira tickets.
+       - Set to false to prevent push request jira tickets and override the
+         associated product's setting.
```

Comment:
`false` will *prevent* Jira tickets?

(This is a copy-paste from Gerrit 447193)

Reply:
Good catch! I've updated the doc